kv/union_store.go

```diff
@@ -60,11 +61,15 @@ func IsErrNotFound(err error) bool {
 type UnionStore struct {
 	Dirty    *memdb.DB // updates are buffered in memory
 	Snapshot Snapshot  // for read
+	TID      uint64    //transactionId
 }
 
 // NewUnionStore builds a new UnionStore.
 func NewUnionStore(snapshot Snapshot) (UnionStore, error) {
+	//dirty := memdb.New(comparer.DefaultComparer, 1 * 1024 * 1024)
```

Comment:
Are you find out why we can read corrupted data from memdb instance which comes from Pool.Get()?